### ISSUE TITLE
Locale Files are Generated with Windows Line-endings

### ISSUE THREAD

**Issue by oleibman** [PHPOffice/PhpSpreadsheet#2172]
This is:

```
- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
Files should have Unix line endings.

### What is the current behavior?
They have Windows line endings. This causes git to think they have all been changed; at least that's what it thinks on my Windows machine (which I have configured to use Unix line endings). It would be easy enough to change them all, but, since they appear to be generated, I'm not sure any such change would stick.

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

### Which versions of PhpSpreadsheet and PHP are affected?
This seems to have been caused by #2111.

**Comment by oleibman** [comment]
As a workaround, I can run `bin/generate-locales` under Cygwin. This changes the (line endings in) the locale files, but git no longer thinks they have been changed. It's weird, but it works.
My guess is that `"\n"` rather than `PHP_EOL` in infra/LocaleGenerator.php will probably figure into the solution, if any is required.

**Comment by oleibman** [comment]
Aha, it all starts to make sense now. There is a locale generator test which runs as part of the test suite, and that's what's causing the files to appear changed. I think I now know the fix, and will start working on it.

**Comment by MarkBaker** [comment]
I'm glad you figured it out.... I test locally against different PHP versions on Windows while I'm working, but I always run the tests on a Linux VM before my commit

**Comment by oleibman** [comment]
Solved by #2174.
